server/src/main/java/org/loomdev/loom/entity/player/PlayerImpl.java

```diff
@@ -214,6 +216,14 @@ private void updateTabList() {
         ));
     }
 
+    private void updateTabListName() {
+        for (ServerPlayer player : getMinecraftEntity().level.getServer().getPlayerList().getPlayers()) {
+            player.connection.send(new ClientboundPlayerInfoPacket(
```

Comment:
I would make one instance of the packet outside of this loop and then send it within the loop. No real reason to make a new one for each player.